resources/sass/abstracts/_variables.scss

```diff
@@ -26,5 +26,21 @@ $color-white: rgb(255, 255, 255);
 $color-white-gray: rgb(235, 235, 236);
 $color-black-gray: rgb(77, 77, 79);
 $color-black: rgb(35, 31, 32);
+$color-black-light: rgb(45, 38, 40);
+$color-black-true: rgb(0, 0, 0);
 
 $transition-duration: 0.3s;
+
+$page-margins: 1600px;
```

Comment:
Šis max būs 1280 px, nosaukums liekas maldinošs, drīzāk nevis margins, bet content-max-width